src/collections/_documentation/workflow/notifications/workflow.md

```diff
@@ -0,0 +1,14 @@
+---
+title: 'Workflow'
+sidebar_order: 2
+---
+
+Most activity within Sentry will generate a workflow notification. Core workflow notifications include:
+
+-   Assignment
+-   Comments
+-   Regressions – when Sentry changes state of an issue from **resolved** to **unresolved**
+
+By default, these notifications are sent to anyone who is a member of the project the issue occurs in, but each member may choose their participation state, as well as opt-in or opt-out of specific issues. 
```

Comment:
"...as well as opt-in or opt-out of specific issues."

Suggestion: "...as well as opt in or opt out of notifications for specific issues (in the "Notifications" section at the bottom right of the issue detail view)."